podcasts/EpisodeDetailViewController+Actions.swift

```diff
@@ -51,10 +51,16 @@ extension EpisodeDetailViewController {
             if !PlaybackManager.shared.playing() {
                 dismiss(animated: true, completion: nil)
             }
+            if let timestamp = timestamp {
```

Comment:
Unsure if we should do this is this is the current playing episode. What do you think @leandroalonso ?

Reply:
Hmm good question! I'd put that into the edge case basket but it might be easy enough to check if it's the current playing episode?